test/interval_tests/complex.jl

```diff
@@ -25,10 +25,12 @@
     @test isequal_interval(interval(0, 1 + im), interval(0 + 0im, 1 + im))
 end
 
-@testset "Inverse roots of unity" begin
-    for i ∈ 0:99
-        @test issubset_interval(cispi( -interval(i)/interval(50) ), inv(cispi( interval(i)/interval(50) ))) &&
-            radius( inv(cispi( interval(i)/interval(50) )) ) < 10eps()
+if (Int != Int32 && VERSION < v"1.10") || VERSION ≥ v"1.10"
```

Comment:
The reason is that `sinpi`/`cospi` from MPFR were only introduced in Julia v1.10 and I do not think this will ever be backported to v1.9.

You are probably right, it should be `@test_broken`.

Reply:
Yes, but in those cases, the implementation should ideally go to some fallback method. Since those tests are not tightness tests (I believe), they should work for a fallback.

Reply:
True. In these cases we could have the default `sinpi(x) = sin(x * pi)` and similarly for `cospi`.

Reply:
That needs to be an interval version of pi.